lms/djangoapps/commerce/__init__.py

```diff
@@ -13,7 +13,19 @@ def create_tracking_context(user):
     }
 
 
+def is_commerce_service_configured():
+    """
+    Return a boolean indicating whether or not configuration is present to use
+    the external commerce service.
+    """
+    return settings.ECOMMERCE_API_URL is not None and settings.ECOMMERCE_API_SIGNING_KEY is not None
```

Comment:
Try `settings.ECOMMERCE_API_URL and settings.ECOMMERCE_API_SIGNING_KEY` in case the values are empty strings.